### ISSUE TITLE
Javascript - How this course will work: Rework The path section

### ISSUE THREAD

**Issue by UniquePixels** [TheOdinProject/curriculum#27341]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

I believe there is some holdover from a previous version of TOP when these were in a different order.  I think two things need to be addressed:
1) Remove references to Ruby; there is no reason to have this verbiage here since we do not ask folks to do Ruby if doing JS.
2) The second paragraph indicates a final project using all the courses in the curriculum, but there are four courses after this one.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript

### (Optional) Discord Name

fabulousgeek

**Comment by JoshDevHub** [comment]
This isn't a holdover from the past: everything you've said is accurate to the way the JavaScript course works within the Ruby on Rails path currently.

Now of course what's strange about this is that if you're doing the JavaScript/Node path, these things are inaccurate and misplaced. I think we can either make this content more agnostic to the context of the overall paths, or we could have two versions of this introduction lesson: one for people in the NodeJS path and one for people in the Rails path.

Do you have any thoughts on what you think is best? @fabulousgk

**Comment by fabulousgk** [comment]
Well, I feel stupid, lol.  I thought this diverged from ruby; it did not occur to me that this is also part of the ruby track (hence why I only marked the Node track). I think we can make it path agnostic pretty easy.  Something like this perhaps:

> How is this course set up? It starts with a deep look at JavaScript organization and then moves through computer science topics, finishing with a major project that integrates everything you have learned so far to create a portfolio-worthy front-end project.

We won’t be focusing deeply on the really basic coding items, so it will move quickly. You should, however, already have completed the [Foundations course](https://www.theodinproject.com/paths/foundations/courses/foundations) – specifically, the [JavaScript Basics section ](https://www.theodinproject.com/paths/foundations/courses/foundations#javascript-basics)– before starting this course.

**Comment by JoshDevHub** [comment]
No need to feel stupid. We rubyists are oft-forgotten :sweat_smile: 

Your suggestion feels mostly okay to me except for the bit about computer science. In the Ruby on Rails path, the computer science content is done as a part of the Ruby course, and that content is absent entirely from the Ruby/Rails path's JavaScript course (you can see what the JS path looks like for Ruby pathers [here](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/javascript))

I had forgotten about this. I also originally thought making it path agnostic was the way to go, but with an entire section being different, maybe it'd be better to just bite the bullet and make two different introductions for each path. I'll think about this some more and discuss with other maintainers and get back to you.

**Comment by JoshDevHub** [comment]
Okay how about this @fabulousgk 

We keep one lesson and we use detail + summary elements as done in the installation lessons for Linux vs. MacOS guides. A [quick example](https://github.com/TheOdinProject/curriculum/blob/main/foundations/javascript_basics/installing_nodejs.md?plain=1#L20) of some code that's using that if you need it.

I think we can move what's currently there into a "Ruby on Rails path" section and the your suggestions can fit in a "NodeJS path" section. Sound okay to you?

**Comment by fabulousgk** [comment]
Yeah, that is great!

**Comment by JoshDevHub** [comment]
Awesome I'll assign you to work on it. Let me know if you have any questions

**Comment by fabulousgk** [comment]
Having an issue with the links.  They don't work inside the HTML.  Do I just convert them to HTML?

**Comment by JoshDevHub** [comment]
Markdown links should work  if you have add the `markdown="block"` attribute to the `details` element. So like
```md
<details markdown="block">
  <summary class="dropDown-header">NodeJS Path</summary>
  
  [this should work](https://theodinproject.com)
</details>
```
If it doesn't, that's okay. You'll definitely be able to use anchor tags and we'll figure out what's going on when you make the PR.

**Comment by fabulousgk** [comment]
Yeah, was just VSCode preview, took your comment to make me realize it would be different.  Works fine in preview tool.  Thanks!
